src/Estimators/WalkerLogInput.h

```diff
@@ -0,0 +1,46 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2024 QMCPACK developers.
+//
+// File developed by: Jaron T. Krogel, krogeljt@ornl.gov, Oak Ridge National Laboratory
+//
+//////////////////////////////////////////////////////////////////////////////////////
+#ifndef QMCPLUSPLUS_WALKERLOGINPUT_H
+#define QMCPLUSPLUS_WALKERLOGINPUT_H
+
+#include "InputSection.h"
+
+namespace qmcplusplus
+{
+
+/** Native representation for walker logs input
+ */
+struct WalkerLogInput : public InputSection
```

Comment:
Noticed a separate issue to be dealt later.
@PDoakORNL since `WalkerLogInput` lives under `QMCDrivers` and Estimators live under `Estimators`, the commonly used bits should no longer stay under `Estimators`, can we relocate `InputSection` under `io`?

Reply:
I think that makes sense.

Reply:
I will add that into my estimator merging work as I have many dependencies on that.